### ISSUE TITLE
GCP build is failing due to missing google-compute-engine package

### ISSUE THREAD

**Issue by vpnachev** [gardenlinux/gardenlinux#253]
**What happened**:
I tried to build GCP image from the main branch (bf4bb69c61f81189a9388c8a23b94591710b9cec) but it is failing with
```bash
$ make gcp 
...
debootstrap --force-check-gpg --variant=minbase --no-merged-usr --keyring=/tmp/tmp.TULKPc8rus/debian-archive-bullseye-keyring.gpg --arch=amd64 --include=auditd,ca-certificates,dnsutils,dracut,iproute2,iptables,iputils-ping,libnss-resolve,libnss-systemd,libpam-systemd,lsb-release,net-
tools,netbase,openssh-server,policykit-1,procps,sudo,sysstat,systemd,systemd-coredump,systemd-cron,traceroute,wget,less,tcpdump,vim-tiny,quota,netcat,fdisk,grub-cloud-amd64,cloud-guest-utils,libpam-cracklib,apparmor,containerd,containernetworking-plugins,docker.io,ethtool,ipvsadm,soc
at,curl,logrotate,arptables,ebtables,xfsprogs,btrfs-progs,jq,ipvsadm,cifs-utils,nfs-common,google-compute-engine,rng-tools-debian --exclude=gcc-9-base,irqbalance bullseye rootfs http://45.86.152.1/gardenlinux/archive/debian/20210527T000000Z
I: Retrieving InRelease
I: Checking Release signature
I: Valid Release signature (key id 0146DC6D4A0B2914BDED34DB648ACFD622F3D138)
I: Retrieving Packages
I: Validating Packages
I: Resolving dependencies of required packages...
I: Resolving dependencies of base packages...
I: Checking component main on http://45.86.152.1/gardenlinux/archive/debian/20210527T000000Z...
E: Couldn't find these debs: google-compute-engine

error: 'debootstrap' failed!

  Full command:

    debootstrap --force-check-gpg --variant=minbase --no-merged-usr --keyring=/tmp/tmp.TULKPc8rus/debian-archive-bullseye-keyring.gpg --arch=amd64 --include=auditd\,ca-certificates\,dnsutils\,dracut\,iproute2\,iptables\,iputils-ping\,libnss-resolve\,libnss-systemd\,libpam-systemd\,ls
b-release\,net-tools\,netbase\,openssh-server\,policykit-1\,procps\,sudo\,sysstat\,systemd\,systemd-coredump\,systemd-cron\,traceroute\,wget\,less\,tcpdump\,vim-tiny\,quota\,netcat\,fdisk\,grub-cloud-amd64\,cloud-guest-utils\,libpam-cracklib\,apparmor\,containerd\,containernetworking
-plugins\,docker.io\,ethtool\,ipvsadm\,socat\,curl\,logrotate\,arptables\,ebtables\,xfsprogs\,btrfs-progs\,jq\,ipvsadm\,cifs-utils\,nfs-common\,google-compute-engine\,rng-tools-debian --exclude=gcc-9-base\,irqbalance bullseye rootfs http://45.86.152.1/gardenlinux/archive/debian/20210
527T000000Z
...
```

**What you expected to happen**:
Building GCP image to succeed.

**How to reproduce it (as minimally and precisely as possible)**:
Run `make gcp`.

**Anything else we need to know**:

**Environment**:

**Comment by vpnachev** [comment]
The build is failing due to https://tracker.debian.org/news/1239663/google-compute-image-packages-removed-from-testing/

**Comment by marwinski** [comment]
@gehoern: as far as I understand this is fixed!? Can we close this ticket or can you add a status if not.

**Comment by nanory** [comment]
Due to issue #549, the mentioned bug should not exist anymore. It may be that the bug has already been fixed before #549, but by completing the mentioned issue, the `google-compute-engine` package is definitely no longer required during the build, because all files have been migrated to the `features/gcp` mechanism. 

In addition, the referenced issue and its pull requests have shown, that the current GCP build does not show any problems, so the problem seems to be fixed.
